rpc/gnoi/gnoi_cert.proto

```diff
@@ -0,0 +1,378 @@
+// This file defines the gNOI API to be used for certificate installation and
+// rotation. go/gnoi-cert
+syntax = "proto3";
+
+package gnoi.certificate;
+
+// The Certificate Management Service exported by targets.
+// The service primarily exports two main RPCs, Install & Rotate which are used
+// for installation of a new certificate, and rotation of an existing
+// certificate on a target, along with a few management related RPCs.
+service CertificateManagementService {
+
+  // Rotate will replace an existing Certificate on the target by creating a
+  // new CSR request and placing the new Certificate based on the CSR on the
+  // target. If the stream is broken or any steps in the process fail the
+  // target must rollback to the original Certificate.
+  //
+  // The following describes the sequence of messages that must be exchanged
+  // in the Rotate() RPC.
+  //
+  // Sequence of expected messages:
+  // Case 1: When Target generates the CSR.
+  //
+  //   Step 1: Start the stream
+  //     Client <---- Rotate() RPC stream begin ------> Target
+  //
+  //   Step 2: CSR
+  //     Client -----> GenerateCSRRequest() ----> Target
```

Comment:
Should these have `()` after them, since they seem to be the submessages of the `RotateCertificateRequest` rahter than RPC which they seem to read like right now?